iocage_lib/ioc_common.py

```diff
@@ -393,10 +393,16 @@ def sort_release(releases, split=False, fetch_releases=False):
 
     if split:
         for i, rel in enumerate(releases):
-            rel, r_type = rel.properties["mountpoint"].value.rsplit("/")[
-                -1].split("-", 1)
+            try:
+                rel, r_type = rel.properties["mountpoint"].value.rsplit("/")[
+                    -1].split("-", 1)
+            except ValueError:
```

Comment:
How can you get ValueError here?

Reply:
When the release is named something like `r323637M`.